Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+
+
+           
+            string firstName = returnString("Enter First Name");
+            string lastName = returnString("Enter Last Name");
+            int age = returnInt("Enter Your Age");
+            float weight = returnFloat("Enter Your Weight in KG");
+            float height = returnFloat("Enter Your Height in M");
+            float bmi = calculateBMI(weight, height);
+            Console.WriteLine("Your BMI is " + bmi);
+            
+            
+
+
+
+
+         
+        }
+
+
+        public static int returnInt(string message)
+        {
+            Console.WriteLine(message);
+            return Convert.ToInt32(Console.ReadLine());
+        }
+
+        public static string returnString(string message)
```

Comment:
You don't just return a string. You `PromptString`. Please rename. Don't use a word `return` in a name ever, it does not add any value.

Reply:
done, thanks for reviewing the work.